### PR TITLE
Add camel-hl7 wrapper

### PR DESCRIPTION
**Pull request by ozgurkisir-qlik**
Add camel hl7 wrapper

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -935,12 +935,12 @@
...
-<!--    <feature name='camel-hl7' version='${project.version}' start-level='50'>-->
-<!--        <feature version='${project.version}'>camel-netty</feature>-->
-<!--        <bundle dependency='true'>mvn:org.apache.mina/mina-core/${mina-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:ca.uhn.hapi/hapi-osgi-base/${hapi-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-hl7/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-hl7' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-netty</feature>
+        <bundle dependency='true'>wrap:mvn:org.apache.mina/mina-core/${mina-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:ca.uhn.hapi/hapi-osgi-base/${hapi-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: This version of library doesn't exist

Comment (ozgurkisir-qlik) [reply features/src/main/feature/camel-features.xml]: Updated

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -935,12 +935,12 @@
...
-<!--    <feature name='camel-hl7' version='${project.version}' start-level='50'>-->
-<!--        <feature version='${project.version}'>camel-netty</feature>-->
-<!--        <bundle dependency='true'>mvn:org.apache.mina/mina-core/${mina-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:ca.uhn.hapi/hapi-osgi-base/${hapi-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-hl7/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-hl7' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-netty</feature>
+        <bundle dependency='true'>wrap:mvn:ca.uhn.hapi/hapi-base/${hapi-version}</bundle>
+        <bundle dependency="true">mvn:org.apache.mina/mina-core/${mina-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]:
```suggestion
        <bundle dependency='true'>mvn:org.apache.mina/mina-core/${mina-version}</bundle>
```

**Review by essobedo — APPROVED** [review]
LGTM

**Pull request merged**